python/prophet/forecaster.py

```diff
@@ -88,6 +89,7 @@ def __init__(
             daily_seasonality='auto',
             holidays=None,
             seasonality_mode='additive',
+            holiday_mode=None,
```

Comment:
Could we place this parameter at the end so that the change doesn't affect people who are using positional keywords? Unfortunate i know.

Reply:
Hadn't even considered that, but I wouldn't want to be on the other side of that kind of change. Done!